magenta/music/musicxml_parser.py

```diff
@@ -395,6 +405,11 @@ def _parse(self):
         self.notes.append(note)
         # Keep track of current note as previous note for chord timings
         self.state.previous_note = note
+
+        # Sum up the MusicXML durations in voice 1 of this measure
```

Comment:
will this be inaccurate if voice 1 doesn't have any notes? would it be better to determine the duration based on the last note the measure of any voice?